index.html

```diff
@@ -86,5 +86,197 @@ <h1>I'm Stephen Glad to see you</h1>
         </div>
       </section>
     </main>
+    <section class="project-section">
+      <div class="project-container flex column">
+        <div class="project-card flex column">
+          <img src="./images/Snapshoot Portfolio.svg" alt="project card image">
+          <div class="prject-card-info flex column">
+            <h2 class="project-title">Tonic</h2>
+            <div class="project-meta flex">
+              <p class="bold">Canopy</p>
+              <div class="round"></div>
+              <p>Backend</p>
+              <div class="round"></div>
+              <p>2015</p>
+            </div>
+            <div class="project-note">
+              <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div class="project-tag">
+              <ul class="project-skills flex">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JAVASCRIPT</li>
+              </ul>
+            </div>
+            <button>See Project</button>
+          </div>
+        </div>
```

Comment:
- [ ] Kindly remove the `background-image` here to match the design in Figma 👍 
- [ ] Also please match the `background-color` of the page which is `#E5E5E5` then the project cards should be white `#fff` 💯 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/205632832-429fa2f9-cffe-45b3-a8e0-fd71c9b59c56.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/205633008-c07e23f9-39ca-48dd-9668-09a92fde37db.png)